packages/universal-components/src/RangeDatePicker/DayItemArrow.js

```diff
@@ -0,0 +1,97 @@
+// @flow
+
+import * as React from 'react';
+import { Animated, Easing, View, Platform } from 'react-native';
+
+import { type StylePropType, StyleSheet } from '../PlatformStyleSheet';
+import { designTokens } from '../DesignTokens';
+
+type Props = {|
+  +style?: StylePropType,
+|};
+
+const SHARED_ANIMATION_CONFIG = {
+  duration: designTokens.durationArrowScaleAnimation,
+  useNativeDriver: true,
+  easing: Easing.ease,
+};
+
+const isAnimationEnabled = Platform.OS !== 'android';
+
+export default class DayItemArrow extends React.Component<Props> {
+  componentDidMount() {
+    this.animating = true;
+    if (isAnimationEnabled) {
+      this.handleAnimation();
```

Comment:
I didn't investigated it further. But it will be probably worth checking.
`handleAnimation` is started twice on initial render. It doesn't seems to be a problem of this component, but for some reason `RenderDay` is mounted, unmounted and mounted again after opening the picker.

But we can definitely merge this one and investigate it (if necessary) in different PR.

Reply:
I think it should be because I'm using this component twice in RenderDay

Reply:
It is fired twice for each arrow, so that's not a problem, but this is just a side effect. As I mentioned, source of this is actually that `RenderDay` is re-mounted after picker opening. But maybe there is some reason for it.